lib/ci.js

```diff
@@ -22,7 +23,8 @@ const CI_TYPES = new Map([
   [LIBUV, { name: 'libuv', re: /libuv/ }],
   [NOINTL, { name: 'No Intl', re: /nointl/ }],
   [V8, { name: 'V8', re: /node-test-commit-v8/ }],
-  [LINTER, { name: 'Linter', re: /node-test-linter/ }]
+  [LINTER, { name: 'Linter', re: /node-test-linter/ }],
+  [LITE, { name: 'Lite Commit CI', re: /node-test-commit-lite/ }]
```

Comment:
Shouldn't this be `/node-test-commit-light/`?

Reply:
Thanks for the review, i got that fixed locally with tests.